pypeit/waveimage.py

```diff
@@ -134,9 +147,43 @@ def __repr__(self):
         txt = '<{:s}: >'.format(self.__class__.__name__)
         return txt
 
+    def save(self, outfile=None, overwrite=True):
+        """
+        Save the master wavelength image.
+
+        Args:
+            outfile (:obj:`str`, optional):
+                Name for the output file.  Defaults to
+                :attr:`file_path`.
+            overwrite (:obj:`bool`, optional):
+                Overwrite any existing file.
+        """
+        super(WaveImage, self).save(self.mswave, 'WAVE', outfile=outfile, overwrite=overwrite,
+                                    steps=self.steps)
+
+    # TODO: it would be better to have this instantiate the full class
+    # as a classmethod.
+    def load(self, ifile=None, return_header=False):
+        """
+        Load the wavelength image data from a saved master frame.
+
+        Args:
+            ifile (:obj:`str`, optional):
+                Name of the master frame file.  Defaults to
+                :attr:`file_path`.
+            return_header (:obj:`bool`, optional):
+                Return the header
+
+        Returns:
+            tuple: Returns an `numpy.ndarray`_ with the wavelength
+            image.  Also returns the primary header, if requested.
+        """
+        return super(WaveImage, self).load('WAVE', ifile=ifile, return_header=return_header)
 
+    
 
 
+# TODO: Provide a staticmethod
```

Comment:
I agree a static method is a more elegant approach.

Reply:
I had missed this one.  Added now.